setup.py

```diff
@@ -18,6 +22,73 @@
 
 name = environ.get("LIBRARY_NAME", "reco_utils")
 
+install_requires = [
```

Comment:
![image](https://user-images.githubusercontent.com/9027725/111640749-dfa5d080-87d2-11eb-968f-066b384d4349.png)
Perhaps like this.

Reply:
i think.... setup_requires is a pre-pip install step?

Reply:
They say they discourage this (also in PEP):
https://setuptools.readthedocs.io/en/latest/references/keywords.html?highlight=setup_requires#keywords